src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimantResponseAgreedSettledPartAdmitDefendantLipNotificationHandler.java

```diff
@@ -49,33 +52,40 @@ public List<CaseEvent> handledEvents() {
 
     @Override
     public Map<String, String> addProperties(CaseData caseData) {
-        return Map.of(
-            RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1()),
-            CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
-        );
+        if (caseData.isRespondent1NotRepresented()) {
+            return Map.of(
+                RESPONDENT_NAME, getPartyNameBasedOnType(caseData.getRespondent1()),
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference()
+            );
+        } else {
+            return Map.of(
+                CLAIM_REFERENCE_NUMBER, caseData.getLegacyCaseReference(),
+                CLAIM_LEGAL_ORG_NAME_SPEC, organisationDetailsService.getRespondentLegalOrganizationName(caseData)
+            );
+        }
     }
 
     private CallbackResponse notifyDefendantForPartAdmitClaimSettled(CallbackParams callbackParams) {
         CaseData caseData = callbackParams.getCaseData();
-        if (!caseData.isLRvLipOneVOne() || caseData.getRespondent1().getPartyEmail() == null) {
-            return AboutToStartOrSubmitCallbackResponse.builder()
-                .build();
+        if (Objects.nonNull(caseData.getRespondent1Email())) {
+            notificationService.sendMail(
+                caseData.getRespondent1Email(),
+                setUpEmailTemplate(caseData),
+                addProperties(caseData),
+                String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
+            );
         }
-
-        notificationService.sendMail(
-            caseData.getRespondent1().getPartyEmail(),
-            setUpEmailTemplate(caseData),
-            addProperties(caseData),
-            String.format(REFERENCE_TEMPLATE, caseData.getLegacyCaseReference())
-        );
-
         return AboutToStartOrSubmitCallbackResponse.builder()
             .build();
     }
 
     private String setUpEmailTemplate(CaseData caseData) {
-        return caseData.isRespondentResponseBilingual()
-            ? notificationsProperties.getRespondentLipPartAdmitSettleClaimBilingualTemplate()
-            : notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate();
+        if (caseData.isRespondent1NotRepresented()) {
+            return caseData.isRespondentResponseBilingual()
+                ? notificationsProperties.getRespondentLipPartAdmitSettleClaimBilingualTemplate()
+                : notificationsProperties.getRespondentLipPartAdmitSettleClaimTemplate();
+        } else {
```

Comment:
you dont need else when you are returning in the if statement. so it would be : 
    if(condition) {
         return something;
  }
return somethingDifferent;
```suggestion
        } else {
```
```suggestion
        } else {
```